aurora/aurora-war/src/main/webapp/look/jsp/TopBar.jsp

```diff
@@ -25,6 +27,8 @@
 </c:if>
 <c:url var="urlLogin" value="/Login"/>
 
+<c:set var="isDisplayDirectory" value="${silfn:isDefined(directoryURL) && ((isAnonymous && settings.displayDirectoryForAnonymous) || not isAnonymous)}"/>
```

Comment:
You should use here 'and' or 'or' operators instead of '&&' or '||'

Reply:
Can guest users access the directory?
I think the answer is no, due to changes in the [silverpeas-header-part.jsp](https://github.com/Silverpeas/Silverpeas-Core/pull/1258/files#diff-a086f345ae8e61dda75e342d07602b9bff4e810085ed7cb0fa12e7adafb38020) file which is the header producer of the integrated look.